content/docs/learning-guide/tutorials/rendering/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,846 @@
+
+1. Towards the end of the function, right before `OUT.m_worldPosition = worldPosition.xyz;`, add the following. This adjusts the object's position in the positive x direction by `5` units.
+   
+   ```hlsl
+   worldPosition.x += 5.0;
+   ```
+   {{< tip >}}
+   You may wonder why you are editing `worldPosition` instead of `m_position`; `m_position` is the position of this vertex relative to the origin of the model, whereas `worldPosition` is the position of this vertex relative to the origin of the level (or world). Try editing the other dimensions of  `m_position` and `worldPosition` and see what they do!
+   {{< /tip >}}
+
+1. Make sure the **Editor** is open, if it is not already open.
+
+1. Save your file with **Ctrl-S** and the **Asset Processor** should automatically detect changes and process the file. You can open the Asset Processor and check when the file is done processing. 
+
+   {{< note >}}
+   If you can't find the Asset Processor, navigate to the Windows taskbar at the bottom right, and click on {{< icon "asset-processor.svg" >}}.
+   {{< /note >}}
+
+1. When the Asset Processor is done processing the changes, you should see in the Editor that your material looks different!
+
+{{< image-width src="/images/atom-guide/vegetation-bending-tutorial/offset.png" width="100%" alt="The shader ball in the Editor, with the offset applied to the forward pass." >}}
+
+The main texture of the shader ball shows up at an offset as intended, but a grey outline is still at the origin of the object. This is because you only edited the forward pass, and have yet to edit the *depth pass*. All the passes this material goes through are referenced in `VegetationBending.materialtype`.
+Keep in mind that different passes render different parts of the material, and some passes' outputs are used as inputs to other passes. You can find more information about passes in the [Passes](/docs/atom-guide/dev-guide/passes/) section.
+
+Repeat the above steps for the depth pass:
+1. Open `VegetationBending_DepthPass.azsli`.
+
+   {{< caution >}}
+   Make sure you are not editing the `VegetationBending_DepthPass_WithPS.azsli` file.
+   {{< /caution >}}
+
+1. Find the function `DepthPassVS`.
+
+1. Towards the end of the function, right before `OUT.m_position = mul(ViewSrg::m_viewProjectionMatrix, worldPosition);`, add: 
+   
+   ```
+   worldPosition.x += 5.0;
+   ```
+
+1. Save your file and look at the Editor. The shader ball should now be completely rendered at an offset! 
+   
+   {{< note >}}
+   Note that the shadow is still in the original position. This is because this tutorial doesn't cover the vertex-only shadowmap shader and instead using the StandardPBR one for brevity. You will be adding a custom shadowmap with pixel shader later on in the tutorial. However, after finishing this tutorial, see if you can add and adjust the vertex-only shadow (used for opaque materials that don't have alpha-cutout) yourself! 
```

Comment:
I'm confused by the mention of the "vertex-only" shadowmap shader. I'm not sure if it's needed here. Similarly, I think that a "vertex-only shadow (used for opaque materials that don't have alpha-cutout)" needs more context for the suggestion to adjust it to be actionable. 

In retrospect, using the term "vertex-only" may make sense as a way to identify something. But right now and at this part of the doc, it's not clear to me what you're referring to. 

I think the core information to call out here is: 
>  Note that the shadow is still in the original position. That's because you haven't updated the shadowmap shader, yet. Later in the tutorial, you will add a custom shadowmap with a pixel shader, which will fix the shadow.

My suggestion below is only to rephrase, doesn't include anything above.
```suggestion
   Note that the shadow is still in the original position. This is because this tutorial doesn't cover the vertex-only shadowmap shader and instead uses the StandardPBR one for brevity. Later in the tutorial, you will add a custom shadowmap with a pixel shader. However, after finishing this tutorial, see if you can add and adjust the vertex-only shadowmap shader yourself! Recall that the vertex-only shadowmap shader is used for opaque materials that don't have alpha-cutout.
```

Reply:
I think you're right, I'm introducing some new terminology that at this point the user probably wouldn't understand (or care about) yet, so I went ahead and just put in what you noted as the important information.